test/testdata/lsp/hover_ampersand_operations.rb

```diff
@@ -18,13 +18,6 @@ def main
                     # ^ hover: sig {returns(String)}
 
   # Safenav
-  dog = Dog.new
-  breed = dog&.breed
```

Comment:
Let's update the test rather then deleting it?

Reply:
Mh... but this test is for hover on ampersand operations... and this case no longer requires an ampersand operation (which results in an error, instead of hover information)

Should we have a different test somewhere else that test for an error being shown instead?

Reply:
> (which results in an error, instead of hover information)

it should have _both_ an error and support hover on it.

Reply:
Done!